docker/nginx/libs/skynet/account.lua

```diff
@@ -76,9 +76,16 @@ function _M.get_account_limits()
 
     if ngx.var.account_limits == "" then
         local httpc = require("resty.http").new()
+        local uri = "http://10.10.10.70:3000/user/limits"
+
+        -- include skylink if it is available in the context of request
+        -- todo: this should not rely on skylink variable to be defined
+        if ngx.var.skylink ~= nil and ngx.var.skylink ~= "" then
+            uri = uri .. "/" .. ngx.var.skylink
```

Comment:
1. What will happen if `ngx.var.skylink` is not defined?
2. Can't we just initialise it with "" if it's not initialised? The trailing slash will be stripped in accounts and the result will be the same.
3. You have very interesting notes in the description of this PR. It would be nice to have them in the code as well, so they are readily available to any dev who wants to change this.

Feel free to resolve this and merge right away - none of this is blocking.

Reply:
I'll follow up with a pull request refactoring this function so skylink is passed as an arg and not implicitly assumed it's defined in a variable.